CHANGELOG.md

```diff
@@ -1,5 +1,144 @@
+
+### Rule changes
+
+- Fix RUF027 false positives if `gettext` is imported using an alias ([#12025](https://github.com/astral-sh/ruff/pull/12025))
+- Update `trapz` and `in1d` deprecation for NPY201 ([#11948](https://github.com/astral-sh/ruff/pull/11948))
+- Modify diagnostic ranges for shell-related bandit rules ([#10667](https://github.com/astral-sh/ruff/pull/10667))
+
+### Server
+- Closing an untitled, unsaved notebook document no longer throws an error ([#11942](https://github.com/astral-sh/ruff/pull/11942))
+- Support the usage of tildes and environment variables in `logFile` ([#11945](https://github.com/astral-sh/ruff/pull/11945))
+- Add option to configure whether to show syntax errors ([#12059](https://github.com/astral-sh/ruff/pull/12059))
+
+### Bug fixes
+
+- Avoid `E203` for f-string debug expression ([#12024](https://github.com/astral-sh/ruff/pull/12024))
+- Consider 2-character EOL before line continuation ([#12035](https://github.com/astral-sh/ruff/pull/12035))
+- Consider line continuation character for re-lexing ([#12008](https://github.com/astral-sh/ruff/pull/12008))
+- Match import name ignores against both name and alias ([#12033](https://github.com/astral-sh/ruff/pull/12033))
+- Use `TokenSource` to find new location for re-lexing ([#12060](https://github.com/astral-sh/ruff/pull/12060))
+- [`pyflakes`] Detect assignments that shadow definitions (`F811`) ([#11961](https://github.com/astral-sh/ruff/pull/11961))
+- Add syntax error for empty type parameter list ([#12030](https://github.com/astral-sh/ruff/pull/12030))
+- Avoid consuming newline for unterminated string ([#12067](https://github.com/astral-sh/ruff/pull/12067))
+- Do not include newline for unterminated string range ([#12017](https://github.com/astral-sh/ruff/pull/12017))
+- Use correct range to highlight line continuation error ([#12016](https://github.com/astral-sh/ruff/pull/12016))
+
+
+### Other changes
+- Upgrade Unicode table ([#11194](https://github.com/astral-sh/ruff/pull/11194))
+- Removed the deprecation error message for the nursery selector ([10172])(https://github.com/astral-sh/ruff/pull/10172)
+
+
+## Release
+- Migrate release workflow to cargo-dist ([#9559](https://github.com/astral-sh/ruff/pull/9559))
+
+### Documentation
+- Clarify special control flow parameters for `PLR0917`: `too-many-positional` ([#11978](https://github.com/astral-sh/ruff/pull/11978))
+- Fix missing related settings header ([#12013](https://github.com/astral-sh/ruff/pull/12013))
+- Update PEP reference in future_rewritable_type_annotation.rs ([#11985](https://github.com/astral-sh/ruff/pull/11985))
+- `ruff-check`: update docs for fix_only ([#11959](https://github.com/astral-sh/ruff/pull/11959))
+- Fix link to `python-lsp-server` ([#11980](https://github.com/astral-sh/ruff/pull/11980))
+- Add `and formatter` to CLI startup message ([#12042](https://github.com/astral-sh/ruff/pull/12042))
```

Comment:
```suggestion
```

Reply:
Why?

Reply:
I understood that we didn't usually include documentation sections in our changelogs, since end users don't really need to know about every change to our docs